radio/src/datastructs.h

```diff
@@ -688,13 +688,25 @@ PACK(struct ModuleData {
   int8_t  rfProtocol:4;
   uint8_t channelsStart;
   int8_t  channelsCount; // 0=8 channels
-  uint8_t failsafeMode:7;
+  uint8_t failsafeMode:4;  //only 3 bits neede
+  uint8_t subType:3;
   uint8_t invertedSerial:1; // telemetry serial inverted from standard
   int16_t failsafeChannels[NUM_CHNOUT];
-  int8_t  ppmDelay:6;
-  uint8_t ppmPulsePol:1;
-  uint8_t ppmOutputType:1;     // false = open drain, true = push pull
-  int8_t  ppmFrameLength;
+  union {
+    struct {
+      int8_t  ppmDelay:6;
+      uint8_t ppmPulsePol:1;
+      uint8_t ppmOutputType:1;     // false = open drain, true = push pull
+      int8_t  ppmFrameLength;
+        };
```

Comment:
I didn't explain well, I meant name the struct inside the union, so that we would have ```moduleData.ppm.delay``` here